src/pages/experiments/[experimentId]/data-exploration/index.jsx

```diff
@@ -120,11 +122,12 @@ const ExplorationViewPage = ({
     'Data Management': {
       toolbarControls: <MosaicCloseButton key='remove-button-data-management' />,
       component: (width, height) => (
-        <CellSetsTool
-          experimentId={experimentId}
-          width={width}
-          height={height}
-        />
+        <></>
+        // <CellSetsTool
```

Comment:
Why comment all the plots?

Reply:
Aah sorry, I was cmmenting this out to make development faster. Forgot to uncomment it. Uncommented.